build.sbt

```diff
@@ -20,7 +21,12 @@ scalacOptions ++= Seq(
   // "-Xlint:_,-adapted-args,-type-parameter-shadow,-inaccessible",
   "-Xfatal-warnings")
 
-version := "0.1-SNAPSHOT"
+javacOptions ++= Seq(
+  "-target", "1.8",
+  "-source", "1.8",
+  )
+
+version := Option(System.getenv("VERSION")).getOrElse("0.1-SNAPSHOT")
```

Comment:
Since we plan to use this fat jar now as a release artifact, we have to stop putting `0.1-SNAPSHOT` in the file name. 😅